docs/pages/_error.js

```diff
@@ -93,6 +93,11 @@ export default class Error extends React.Component {
       redirectPath = `https://reactnative.dev/docs/${page}`;
     }
 
+    // Remove versiong from path if the version is still supported, to redirect to the root
```

Comment:
```suggestion
    // Remove version from path if the version is still supported, to redirect to the root
```

Reply:
Woops. I'm not that great with computers, sorry!